src/components/Comments/ReplyCommentItem.js

```diff
@@ -201,10 +201,10 @@ const ReplyCommentItem = ({
             </ButtonHightlight>
           </TouchableOpacity>
         )}
-        <TouchableOpacity onPress={() => onBlock(comment)} testID="btnBlock" activeOpacity={1}>
+        <TouchableOpacity onPress={onBlock} testID="btnBlock" activeOpacity={1}>
           <ButtonHightlight
             onLongPress={handleLongPress}
-            onPress={() => onBlock(comment)}
+            onPress={onBlock}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onPress` prop of `TouchableOpacity` and `ButtonHightlight` both point to the same `onBlock` function. This will result in the `onBlock` function being called twice when the button is pressed. If this is not the intended behavior, consider removing one of them.

```diff
- <TouchableOpacity onPress={onBlock} testID="btnBlock" activeOpacity={1}>
+ <TouchableOpacity testID="btnBlock" activeOpacity={1}>
    <ButtonHightlight
      onLongPress={handleLongPress}
      onPress={onBlock}
      style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}>
      <IconEn name="block" size={15.02} color={colors.gray1} />
    </ButtonHightlight>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->